app/javascript/src/components/icon/Plus.svelte

```diff
@@ -0,0 +1,3 @@
+<svg viewBox="0 0 32 32" width="32px" height="32px" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:svg="http://www.w3.org/2000/svg" class="fill-current-color">
```

Comment:
Since the SVG is inline most of the fluff can be remove. `version`, `xmlns`, `sxmlns:svg` are are not necessary.